op/framework.py

```diff
@@ -12,6 +12,10 @@
 class Handle:
     """Handle defines a name for an object in the form of a hierarchical path.
 
+    Two instances of an object in memory with the same handle path represent the
+    same object in the system. An example of this would be two copies of the same
+    event being delivered to two different observers.
```

Comment:
This makes it sound like people might intentionally have multiple objects in memory and that's part of the design. Nope.. the design is that we try to make this case not create problems for people because it's unavoidable since Python is garbage collected. All of our documentation should instead say it's NOT alright to do that.

Reply:
Well, events are called out specifically because we *do* intentionally create multiple copies at once because of the reason you said during the meeting: that we don't want people to think that they can attach arbitrary attributes to an instance and have it show up in another handler.  But in the general case, I agree that we don't want people doing that without a really good reason.